dbus-opendtu.py

```diff
@@ -33,34 +33,32 @@ def main():
     config.read(f"{(os.path.dirname(os.path.realpath(__file__)))}/config.ini")
     logging_level = config["DEFAULT"]["Logging"].upper()
     dtuvariant = config["DEFAULT"]["DTU"]
-    number_of_inverters = int(config["DEFAULT"]["NumberOfInvertersToQuery"])
 
-    try:
-        number_of_templates = int(config["DEFAULT"]["NumberOfTemplates"])
-    except Exception:
-        number_of_templates = 0
-
-    log_rotate_handler = logging.handlers.RotatingFileHandler(
-        maxBytes=5*1024*1024*10,
-        backupCount=2,
-        encoding=None,
-        delay=0,
-        filename="%s/current.log" % (os.path.dirname(os.path.realpath(__file__)))
-    )
     logging.basicConfig(
         format="%(asctime)s,%(msecs)d %(name)s %(levelname)s %(message)s",
         datefmt="%Y-%m-%d %H:%M:%S",
         level=logging_level,
-        handlers=[
-            logging.StreamHandler(),
-            log_rotate_handler
-        ],
     )
 
+    try:
+        number_of_inverters = int(config["DEFAULT"]["NumberOfInvertersToQuery"])
+    except (KeyError, ValueError) as ex:
+        logging.warning("NumberOfInvertersToQuery: %s", ex)
+        logging.warning("NumberOfInvertersToQuery not set, using default")
+        number_of_inverters = 0
+
+    try:
+        number_of_templates = int(config["DEFAULT"]["NumberOfTemplates"])
+    except (KeyError, ValueError) as ex:
+        logging.warning("NumberOfTemplates: %s", ex)
+        logging.warning("NumberOfTemplates not set, using default")
+        number_of_templates = 0
+
+
     tests.run_tests()
 
     try:
-        logging.info("Start")
+        logging.critical("Start")
```

Comment:
I'm a bit confused. I thought the aim was to keep log entries to a minimum. This seems like the most detailed message possible for the service. I can understand it if I have info logging turned on, but do I really want to see that when the service starts, especially if I don't want any logs at all?

Reply:
agree, but it shows at a minimum level und only at startup, that the service is started and what is registered, and only at startup